perceval/backends/core/discourse.py

```diff
@@ -234,6 +234,9 @@ def __parse_topics_page(self, raw_json):
 
         for topic in topics_page['topic_list']['topics']:
             topic_id = topic['id']
+            # a topic without posts? - acs
```

Comment:
What does this comment mean?

Reply:
Before merging this, we need some unit tests.

Reply:
About the comment, I am not totally sure why the topic does not have a last updated date, so my guess is that it has no posts yet. This is why I have left this comment.

I will work in the unit tests, yes!